src/data-services/scip-backend.js

```diff
@@ -1,68 +1,89 @@
-}
+    const wkt = geoJSONtoWKT(region);
 
-export function getWatersheds(overlap = null, common_name = null, subgroup = null) {
-    let params = {kind: "watershed"};
-    if(overlap) {
-        params.overlap = overlap;
+    if (region && wkt.length > MAXAREALENGTH) {
+        // send a POST reqiest to accomodate long overlap string
+        const formData = new FormData();
+        formData.append('overlap', wkt);
+
+        return axios.post(
+            `${process.env.REACT_APP_SCIP_API_URL}/population`,
+            formData
+        ).then(response => response.data);
     }
-    if(common_name) {
-        params.common_name = common_name;
-        if(subgroup) {
-            params.subgroup = subgroup;
-        }
+    else {
+        //send a GET request
+        let params = {overlap: wkt};
+        return axios.get(
+            `${process.env.REACT_APP_SCIP_API_URL}/population`, { params: params}
+        ).then(response=> response.data);
     }
-
-    return axios.get(
-        process.env.REACT_APP_SCIP_API_URL + "/region",
-        {
-            params: params
-        }
-    )
-    .then(response => response.data);
 }
 
-export function getBasins() {
-    return axios.get(
-        process.env.REACT_APP_SCIP_API_URL + "/region",
-        {
-            params: {kind: "basin"}
+// function that does the work of putting together queries for
+// the "region" API.
+function getRegions(kind, overlap=null, common_name = null, subgroup = null) {
+    const wkt = geoJSONtoWKT(overlap);
+
+    if (overlap && wkt.length > MAXAREALENGTH) {
+        // send a POST reqiest to accomodate long overlap string
+        const formData = new FormData();
+        formData.append('kind', kind);
+        formData.append('overlap', wkt);
+
+        if (common_name) {
+            formData.append('common_name', common_name);
+            if(subgroup) {
+                formData.append('subgroup', subgroup);
+            }
         }
-    )
-    .then(response => response.data);
-}
 
-export function getConservationUnits(overlap = null, common_name = null, subgroup = null) {
-    let params = {kind: "conservation_unit"};
-    if(overlap) {
-        params.overlap = overlap;
+        return axios.post(
+            `${process.env.REACT_APP_SCIP_API_URL}/region`,
+            formData
+        ).then(response => response.data);
     }
-    if(common_name) {
-        params.common_name = common_name;
-        if(subgroup) {
-            params.subgroup = subgroup;
+    else {
+        //send a GET request
+        let params = {kind: kind};
+        if (overlap) {
+            params.overlap = wkt;
         }
+        if (common_name) {
+            params.common_name = common_name;
+            if(subgroup) {
+                params.subgroup = subgroup;
+            }
+        }
+        return axios.get(
+            `${process.env.REACT_APP_SCIP_API_URL}/region`, { params: params}
+        ).then(response=> response.data);
     }
+}
```

Comment:
The pattern of sending a request as either a GET or a POST is common to 4 or 5 data services. I wonder, but am not sure, if it makes sense to factor that out as a single utility function and just pass in the details to it.